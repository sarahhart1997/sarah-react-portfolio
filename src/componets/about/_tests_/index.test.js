import React from 'react';

import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import About from '..';
const { asFragment } = render(<About />);

afterEach(cleanup);

describe('About componet', () => {
    // First Test
    it('renders', () => {
        render(<About />);
    });
    // Second Test
    it('matches snapshot DOM node structure', () => {
        //render About
        expect(asFragment()).toMatchSnapshot();
    });
})