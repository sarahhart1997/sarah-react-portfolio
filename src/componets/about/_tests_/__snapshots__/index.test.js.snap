// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About componet matches snapshot DOM node structure 1`] = `<DocumentFragment />`;
